### PR TITLE
feat(errors): add statusCode to errors

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 language: node_js
 node_js:
-  - stable
-before_install:
-  - npm install -g npm@2 # npm3 buggy
-  - npm prune
+  - 6
```
Comment (bobylito) [inline .travis.yml]: We're finally in for node 6? :)

File: package.json [hunk 2]
```diff
@@ -93,13 +92,12 @@
...
     "morgan": "^1.7.0",
     "mversion": "^1.10.1",
-    "npm-shrinkwrap": "^200.5.1",
     "phantomjs-prebuilt": "^2.1.7",
     "pretty-bytes": "^2.0.1",
     "proxy": "^0.2.4",
     "proxy-agent": "^2.0.0",
     "self-signed": "^1.3.1",
-    "server-destroy": "^1.0.1",
+    "server-destroy-vvo": "^1.0.1",
```
Comment (bobylito) [inline package.json]: -vvo? Only for you?

Comment (vvo) [inline package.json]: https://github.com/isaacs/server-destroy/pull/7 :D

**Comment by vvo** [comment]
@algolia/javascript Tests running on saucelabs will likely fails theses days because of: https://twitter.com/vvoyer/status/750669632961470465

**Pull request merged**